src/Cli.Tests/EndToEndTests.cs

```diff
@@ -631,6 +631,75 @@ public void TestHelpWriterOutput(string command, string flags, string[] expected
 
     }
 
+    /// <summary>
+    /// Test to verify that .env is supported by our CLI.
+    /// Below test is covering the following scenarios:
+    /// 1. Setting up environment variable in the system is consumed correctly.
+    /// 2. .env file if available will use the variable values from the .env file.
+    /// 3. If same variable is defined in both system and .env file, .env file takes precedence.
+    /// 4. If a variable is not present in the .env file then the system defined variable would be used, if defined.
+    /// </summary>
+    [TestMethod]
+    public void TestSupportForEnvironmentVariableFile()
```

Comment:
Looking through this test, it looks like we're testing a few different things here, first that we can read from `System.Environment` accurately, and then another that uses `DotEnv`.

We should split the tests into multiple smaller tests to it's easier to maintain.

Reply:
done